content/faq/how-do-I-use-composer-with-github-access-token.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How do I use Composer with GitHub personal access token?
+menu:
+  main:
+    parent: 'faq'
+    weight: 40
+weight: 40
+linkTitle: Setup GitHub token
+---
+If you need to download private repositories from GitHub during the build
+process, an access token has to be available during `composer install`.
+
+To set up that, make sure you are using one of the Docker images from
+[ContinuousPipe dockerfiles](https://github.com/continuouspipe/dockerfiles)
+as base image. These images are prepared to take a GITHUB_TOKEN 
+build argument which is used to set the token like this:
+`composer global config github-oauth.github.com "$GITHUB_TOKEN"`
+
+Note: Docker build arguments are used only during the build process 
+and will not be included in the built image. However the composer 
+call above will save it in the filesystem of the image. So the final
+Docker images should be pushed into a private repository.
```

Comment:
perhaps use the Warning shortcode:

{{< warning title="Warning" >}}
blah
{{< /warning >}}